lms/djangoapps/course_home_api/outline/v1/views.py

```diff
@@ -129,9 +150,32 @@ def get(self, request, *args, **kwargs):
             'course_tools': course_tools,
             'dates_widget': dates_widget,
             'handouts_html': handouts_html,
+            'welcome_message_html': welcome_message_html,
         }
         context = self.get_serializer_context()
         context['course_key'] = course_key
         serializer = self.get_serializer_class()(data, context=context)
 
         return Response(serializer.data)
+
+
+@api_view(['POST'])
+@authentication_classes((JwtAuthentication,))
+@permission_classes((IsAuthenticated,))
+def dismiss_welcome_message(request):
+    course_id = request.data.get('course_id', None)
+
+    # If body doesnt contain 'course_id', return 400 to client.
```

Comment:
I'm not sure we need these comments here,  I think the code is pretty self-explanatory.